data/human/free worlds 3 reconciliation.txt

```diff
@@ -2406,6 +2407,7 @@ mission "FW Syndicate Extremists 1A"
 	
 	on offer
 		log "With the Pug defeated, it is finally time to deal with the Syndicate extremists and the Alphas that they allied with. The extremists are armed with nuclear missiles, but the Syndicate CEO has agreed to provide a cloaking device that will make it possible to dodge the missiles."
+		log "People" "Alastair Korban" `After the Pug were defeated, Korban admitted the involvement of the Syndicate and the Alphas in the nuclear bombings of Martini and Geminus. As a result, he resigned his position as the CEO of the Syndicate Corporation.`
```

Comment:
He had already admitted the involvement of the Syndicate (well, of "a certain extremist faction that we have since purged from our ranks,") in the bombings when we visited him earlier. And, we had already established it for ourselves prior to the Pug's overt involvement.
After dealing with the Pug, when we visit him, he admits that the Alphas were involved.
Also, the tense is wrong.
How about:
```suggestion
		log "People" "Alastair Korban" `Following the defeat of the Pug, Korban has revealed Alpha involvement and assistance in the bombings of Geminus and Martini. Having overseen the Syndicate as it worked with Alphas to start a civil war in human space, he has resigned from his position as CEO of the Corporation.`
```

Reply:
I'll need to connect the first log entry with the second one. I'll look at it again.